api/render.go

```diff
@@ -47,8 +47,11 @@ func (api *RendererAPI) renderMap(w http.ResponseWriter, r *http.Request) {
 	var bytes []byte
 
 	switch renderType {
-	case "html":
-		bytes, err = renderer.RenderHTML(renderRequest)
+	case "svg":
```

Comment:
is it worth logging to say handling type x ?

Reply:
Adding render_type to the log at the top of the renderMap method